data/human/human missions.txt

```diff
@@ -5547,3 +5547,130 @@ mission "Quicksilver Mixup 1"
+			`	The officer looks back at you, having dusted himself off and regained his senses. "Yes, go and deal with his ship now. We'll have Syndicated Security ships come after you to arrest the hijackers and tug it down. Go!"`
+				launch
+			
+			label securityjob
+			`	The officer, slightly flustered, calls for a small squadron of Syndicated Security ships to chase the hijacked <npc>, then runs off to one of <origin>'s spaceport terminals to file the paperwork with the captain.`
+				decline
+	
+	npc save disable
+		personality launching daring uninterested target
+			confusion 40
+		government "Bounty (Disguised)"
+		ship "Argosy (Blaster)" "Moonshake Express"
+		dialog `You've disabled the <npc>. Syndicated Security will now deal with the hijackers, but you should go back and report to the captain whose ship was hijacked.`
+		to spawn
+			has "argosy hijacking accepted"
+
+	on fail
+		conversation
+			`When you land, the captain of the <npc> comes up to you, his face rather red.`
+			`	He grabs your shoulders and shakes you hard, shouting, "Didn't I tell you to not destroy my ship? Do you know how many credits I've just lost because of you?"`
+			choice
+				`	"I'm sorry, I was just trying to help!"`
+					goto help
+				`	"I'm sure whatever you lost wasn't worth all that much."`
+					goto worthless
+				`	"Well, I hope you had insurance on that ship, at least."`
+					goto insurance
+			label help
+			`	He stops shaking you for a moment to say, "Well, you shouldn't have blown up my <npc model> while you were trying to help, weren't you? Fat lot of help that was, even if you tried to do the right thing. Oh, well..."`
+				goto end
+
+			label worthless
+			`	He looks at you increduleously, scrunching up his face. "Not- not that much? Is that what you just said?" He pauses for a moment to look at the wall above you, looking lost, then he stares you down again with even more intensity. "That was my life's work up there, an <npc model> that had served me for 14 years, and you have the audacity to say it was all for nothing?!"`
+			choice
+				`	"Well, yeah."`
+				`	"No, that's not what I meant."`
+			`	He scoffs at you, and lets you go with a rough push. "I don't need your explanation, I don't think you could give me one. I just can't understand how you can walk away feeling alright that you've set back one man's life so much."`
+				goto end
+			
+			label insurance
+			`	He looks off to his left for just a moment with a look of loss in his eyes, then lets you go with a rough push. "Mmm, no, like a cocksure bastard I sure didn't. Ah, now that one isn't your fault, but if you had just taken your finger off the trigger for a few seconds less, I wouldn't be in this position now, would I? Bastards, you and the hijackers..."`
+
+			label end
+			`	Suddenly, he turns around and walks off towards the direction of the bar, leaving you to contemplate your actions.`
```

Comment:
```suggestion
			`	He lets go and walks off towards the direction of the bar, leaving you to contemplate.`
```

Reply:
In some of the various bits of dialogue, he's already let you go. I'll look through those again and see what I can change, because now I see that in some of them the captain doesn't let you go.